pkg/retriever/reportretriever.go

```diff
@@ -277,6 +284,11 @@ func (r *Retriever) FetchClusters(monitor *monitor.Monitor, input chan types.Man
 				glog.Warningf("Unable to get CRC Token, Using previous Token: %v", err)
 			}
 		}
+		if len(ContentsMap) < 1 {
+			r.InitializeContents(hubID, dynamicClient)
+		} else if len(ContentsMap) > 0 && r.GetContentConfigMap(dynamicClient) == nil {
+			r.CreateInsightContentConfigmap(dynamicClient)
```

Comment:
@zlayne If we have any error in CreateInsightContentConfigmap - Do we still want to continue fetching ? We will not have all the data in PR .

Reply:
If the `CreateInsightContentConfigmap` function fails then the only affected thing is the UI that displays the reason/resolution. Which I am thinking is okay, as we will try to update the configmap content data on the next go round. The PR will not have all the data if the `InitializeContents` func fails, and that has error handling.